### ISSUE TITLE
create-react-app build fails

### ISSUE THREAD

**Issue by zcuric** [BeAnMo/json-find#4]
Hi, 
when trying to create an optimized build with `create-react-app` it fails throwing this error: 
```
Failed to minify the code from this file:

        ./node_modules/json-find/json-find.js:17
```
So this line: 
https://github.com/BeAnMo/json-find/blob/master/json-find.js#L17

**Comment by BeAnMo** [comment]
I appreciate letting me know. This isn't an issue with json-find but with create-react-app's build decisions.

I can make the switch from es6 -> es5, but it's going to have to wait at least until this weekend.

**Comment by zcuric** [comment]
@BeAnMo Yes, you are right. This is fixable by using `react-scripts@next`, so it will fixed in 2.0.0. version.
